.circleci/config.yml

```diff
@@ -121,8 +122,6 @@ workflows:
           name: sdk-e2e-tests
```

Comment:
Let's move it up to be right after sonar

Reply:
done